website/docs/Use-Cases/Zero-Shot-AutoML.md

```diff
@@ -0,0 +1,247 @@
+Also, each estimator may require specific preprocessing of the data. `X_transformed` is the preprocessed data, and `feature_transformer` is the preprocessor. It needs to be applied to the test data before prediction. These are automated when you use the "flamlized" learner. When you use the open-box way, pay attention to them.
+
+### Combine zero shot AutoML and hyperparameter tuning
+
+Zero Shot AutoML is fast. If tuning from the recommended data-dependent configuration is required, you can use `flaml.AutoML.fit()` and set `starting_points="data"`. For example,
+
+```python
+from flaml import AutoML
+automl = AutoML()
+automl_settings = {
+    "task": "classification",
+    "starting_points": "data",
+    "estimator_list": ["lgbm"],
+    "time_budget": 600,
+    "max_iter": 50,
+}
+automl.fit(X_train, y_train, **automl_settings)
+```
+
+Note that if you set `max_iter=0` and `time_budget=None`, you are effectively using zero-shot AutoML. When `estimator_list` is omitted, the estimator together with its hyperparameter configuration will be decided in a zero-shot manner.
+
+### Use your own meta-learned defaults
+
+To use your own meta-learned defaults, specify the path containing the meta-learned defaults. For example,
+
+```python
+estimator = flaml.default.LGBMRegressor(default_location="location_for_defaults")
+```
+
+Or,
+
+```python
+preprocess_and_suggest_hyperparams(
+    "classification", X_train, y_train, "lgbm", location="location_for_defaults"
+)
+```
+
+Or,
+
+```python
+X_train, y_train = load_iris(return_X_y=True, as_frame=as_frame)
+automl = AutoML()
+automl_settings = {
+    "task": "classification",
+    "log_file_name": "test/iris.log",
+    "starting_points": "data:location_for_defaults",
+    "estimator_list": ["lgbm", "xgb_limitdepth", "rf"]
+    "max_iter": 0,
+}
+automl.fit(X_train, y_train, **automl_settings)
+```
+
+Since this is a multiclass task, it will look for the following files under `{location_for_defaults}/`:
+
+- `all/multiclass.json`.
+- `{learner_name}/multiclass.json` for every learner_name in the estimator_list.
+
+Read the next section to understand how to generate these files if you would like to meta-learn the defaults yourself.
+
+## How to Prepare Offline
+
+This section is intended for:
+1. AutoML providers for a particular domain.
+1. Data scientists or engineers who need to repeatedly train models for similar tasks with varying training data.
+
+Instead of running full hyperparmaeter tuning from scratch every time, one can leverage the tuning experiences in similar tasks before. While we have offered the meta-learned defaults from tuning experiences of several popular learners on benchmark datasets for classification and regression, you can customize the defaults for your own tasks/learners/metrics based on your own tuning experiences.
```

Comment:
```suggestion
Instead of running full hyperparameter tuning from scratch every time, one can leverage the tuning experiences in similar tasks before. While we have offered the meta-learned defaults from tuning experiences of several popular learners on benchmark datasets for classification and regression, you can customize the defaults for your own tasks/learners/metrics based on your own tuning experiences.
```